inc/class-activation.php

```diff
@@ -229,12 +229,6 @@ private function wpmuActivate() {
 				'post_type' => 'back-matter',
 				'menu_order' => 1,
 			],
-			'contributors' => [
```

Comment:
No longer automatically creates content with the contributors back matter type when registering a new book.